### ISSUE TITLE
How many fine-tuning steps are required?

### ISSUE THREAD

**Issue by Tonniia** [logn-2024/Any2anyTryon#16]
Hi~Since Flux pre-training using consistent first-dim RoPE, it might be very difficult to encourage first-dim to distinguish different conditions for different tasks.
Could you share how many steps it takes to reach convergence? Thanks!
